frontend/src/lib-customizations/citizen.tsx

```diff
@@ -2,17 +2,36 @@
 //
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
+/// types: node
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
-import customizations from '@evaka/customizations/citizen'
+import defaultsUntyped from '@evaka/customizations/citizen'
```

Comment:
Why have these been renamed? The `untyped` part puzzles me 🤔

Reply:
All customizations follow this basic structure:

```typescript
import defaultsUntyped from '@evaka/customizations/citizen'

const defaults: CitizenCustomizations = defaultsUntyped

const customizations: CitizenCustomizations = overrides
  ? mergeWith({}, defaults, overrides, mergeCustomizer)
  : defaults
```

`defaultsUntyped` has type `any`, because we're importing from an alias that TS compiler can't understand, so we need to then add type for it (and silence eslint warnings from any -> CitizenCustomizations assignment)